server/render.js

```diff
@@ -73,19 +73,29 @@ function render(req, res, next) {
     );
 
     const respond = (error = null) => {
-      const renderResult = error !== TimeoutError;
-
-      const body = renderResult ? renderToString(app) : '';
-      const state = renderResult ? store.getState() : {};
       const helmet = Helmet.rewind();
-
-      return res.send(
-        renderPage({
-          body,
-          state,
-          helmet
-        })
-      );
+      const render = (body = '', state = {}) =>
+        res.send(
+          renderPage({
+            body: '',
+            state: {},
+            helmet
+          })
+        );
+
+      // Skip server side rendering on errors:
```

Comment:
This only applies if the error is a timeout error

Reply:
When we get other errors (eg. 404) we still want to use the SSR render.

Reply:
fixed